src/fixes.rs

```diff
@@ -41,6 +42,7 @@ fn fixlist() -> Vec<Box<dyn Fix>> {
         Box::new(lowercase_key::LowercaseKeyFixer::default()),
         Box::new(space_character::SpaceCharacterFixer::default()),
         Box::new(trailing_whitespace::TrailingWhitespaceFixer::default()),
+        Box::new(ending_blank_line::EndingBlankLineFixer::default()),
```

Comment:
Maybe this fixer should run after all fixers?
@dotenv-linter/core @evgeniy-r What do you think about it?

Reply:
take a look and let me know if this is correct, but I added this to the end of all the other fixes, in the line:

```rust
        // Then we should run the fixers that handle the line entry collection at whole.
        // And at the end we should run the fixer for ExtraBlankLine check (because the previous
        // fixers can create additional extra blank lines).
        Box::new(ending_blank_line::EndingBlankLineFixer::default()),
```